app/controllers/api/thumbnails_controller.rb

```diff
@@ -3,11 +3,11 @@ class ThumbnailsController < ApplicationController
     before_filter :find_uri, only: [:list]
     respond_to :json
 
-    #https://was-thumbnail-dev.stanford.edu/api/v1/was/thumbnails/druid_id/druid:jx731tz7613?format=json
+    #https://was-thumbnail-dev.stanford.edu/api/v1/was/thumbnails/druid_id/jx731tz7613?format=json
     def list
       uri_id = @seed_uri[:id]
       @druid_id = @seed_uri[:druid_id]
-      @memento_records = Memento.where( "uri_id = ? AND is_selected = 1 AND is_thumbnail_captured = 1", uri_id )
+      @memento_records = Memento.where( 'uri_id = ? AND is_selected = 1 AND is_thumbnail_captured = 1', uri_id )
     end
   
   private 
```

Comment:
It might be useful to take advantage of Strong Parameters: http://edgeapi.rubyonrails.org/classes/ActionController/StrongParameters.html
https://github.com/rails/strong_parameters